src/Client.js

```diff
@@ -326,8 +343,25 @@ Client.prototype._execute = function(method, path, data, query, options) {
         endTime
       )
       self._handleRequestResult(response, result, options)
+      
+      const metricsHeaders = [
+        'x-compute-ops',
+        'x-read-ops', 
+        'x-write-ops', 
+        'x-query-time', 
+        'x-txn-retries'
+      ]
 
-      return responseObject['resource']
+      if (options && options.metrics) {
+        return { 
+          value: responseObject['resource'],
```

Comment:
Is there a benefit to returning an object here? What about this usage:

```
const [ response, metrics ] = client.queryWithMetrics(q.Now())
```

If there is value to returning an object, can we call it `response` instead of `value`?

Reply:
`value` was a request to align the JS driver with the JVM driver, which already has a `queryWithMetrics` workflow.